lib/time.js

```diff
@@ -152,17 +152,27 @@ function CronTime(luxon) {
 			}
 
 			if (typeof this.utcOffset !== 'undefined') {
-				let offset =
-					this.utcOffset >= 60 || this.utcOffset <= -60
-						? this.utcOffset / 60
-						: this.utcOffset;
+				if (typeof this.utcOffset === 'string') {
```

Comment:
`utcOffset` "should be an integer amount representing the number of minutes offset (like 120 for +2 hours or -90 for -1.5 hours)"

Reply:
That's weird since the code didn't support that before you implemented it :thinking: was the test passing? could you please point to that test?
But indeed, that might be a good thing to support strings as well, @intcreator wdyt?

Reply:
This was the test case which was passing as well since we are using parseInt

`
it('should run a job using cron syntax with string format utcOffset', () => {
			const clock = sinon.useFakeTimers();
			const callback = jest.fn();
			const luxon = require('luxon');
			// Current time
			const t = luxon.DateTime.local();
			// UTC Offset decreased by an hour (string format '(+/-)HH:mm')
			const utcOffset = t.offset - 60;
			let utcOffsetString = utcOffset > 0 ? '+' : '-';
			utcOffsetString += ('0' + Math.floor(Math.abs(utcOffset) / 60)).slice(-2);
			utcOffsetString += ':';
			utcOffsetString += ('0' + (utcOffset % 60)).slice(-2);

			const job = new cron.CronJob(
				t.second + ' ' + t.minute + ' ' + t.hour + ' * * *',
				callback,
				null,
				true,
				null,
				null,
				null,
				utcOffsetString
			);

			// tick 1 sec before an hour
			clock.tick(1000 * 60 * 60 - 1);
			expect(callback).toHaveBeenCalledTimes(0);

			// tick 1 sec
			clock.tick(1);
			clock.restore();
			job.stop();
			expect(callback).toHaveBeenCalledTimes(1);
		});
`

Reply:
right, `parsetInt` just ignores whatever is after the semicolon ... those kinds of implicit behaviors are really hazardous :sweat_smile:

Reply:
my opinion is that we explicitly support strings. if there was unintended support for it before we want our changes to be backwards compatible. however we should explicitly parse it in a clear way